File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/GetTicketHistoryResponseContent.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import de.dimedis.mobileentry.model.Action;
import de.dimedis.mobileentry.model.Ticket;

public class GetTicketHistoryResponseContent extends BaseResponseContent {
    public static final String ACTIONS = "actions";
    public static final String TICKET_CODE = "ticket_code";
    public static final String TICKET = "ticket";


    @SerializedName(ACTIONS)
    private List<Action> listActions;

    @SerializedName(TICKET_CODE)
    private String ticket_code;

    @SerializedName(TICKET)
    private Ticket mTicket;

    public List<Action> getListActions() {
        return listActions;
    }

    public String getTicketCode() {
        return ticket_code;
    }

    public Ticket getTicket() {
        return mTicket;
    }

    @Override
    public String toString() {
        return "GetTicketHistoryResponseContent{" +
                "listActions=" + listActions +
                ", ticket_code='" + ticket_code + '\'' +
                ", mTicket=" + mTicket +
                '}';
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/GetTicketHistoryResponseContent.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName
import de.dimedis.mobileentry.model.Action
import de.dimedis.mobileentry.model.Ticket

class GetTicketHistoryResponseContent : BaseResponseContent() {
    companion object {
        const val ACTIONS = "actions"
        const val TICKET_CODE = "ticket_code"
        const val TICKET = "ticket"
    }

    @SerializedName(ACTIONS)
    val listActions: List<Action?>? = null

    @SerializedName(TICKET_CODE)
    private val ticket_code: String? = null

    @SerializedName(TICKET)
    private val mTicket: Ticket? = null

    fun listActions(): List<Action?>? {
        return listActions
    }

    fun getTicketCode(): String? {
        return ticket_code
    }

    fun getTicket(): Ticket? {
        return mTicket
    }

    override fun toString(): String {
        return "GetTicketHistoryResponseContent{" +
                "listActions=" + listActions +
                ", ticket_code='" + ticket_code + '\'' +
                ", mTicket=" + mTicket +
                '}'
    }
}